ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/base-fee-input/base-fee-input.js

```diff
@@ -23,7 +19,6 @@ import FormField from '../../../../ui/form-field';
 
 import { useAdvancedGasFeePopoverContext } from '../../context';
 import AdvancedGasFeeInputSubtext from '../../advanced-gas-fee-input-subtext';
-import { renderFeeRange } from '../utils';
```

Comment:
seems like we can delete the `renderFeeRange` method. Seems like you've replaced it with `formatGasFeeOrFeeRange`?

Reply:
Oh lol you did this already.